src/util/useStartsWithChar.ts

```diff
@@ -0,0 +1,21 @@
+import { useEffect, useState } from 'react';
+
+// FPCC
+import { FvCharacter, FvWord } from '../components/common/data';
+
+export const useStartsWithChar = (
+  dictionaryData: FvWord[],
+  character: FvCharacter
+) => {
+  const [entriesStartingWith, setEntriesStartingWith] = useState<FvWord[]>([]);
+  useEffect(() => {
+    if (dictionaryData && character.sortingFormNum !== undefined) {
+      const entries = [...dictionaryData].filter((entry) => {
+        return entry.sorting_form[0] === character.sortingFormNum;
+      });
+      setEntriesStartingWith(entries);
+    }
+  }, [dictionaryData, character.sortingFormNum]);
+
+  return { entriesStartingWith };
+};
```

Comment:
Let's log a ticket to deal with the entries that start with a character that isn't in the alphabet. For the ones that start with punctuation, we might want to count the first non-ignorable character when figuring out what it "starts with"

Reply:
Created https://firstvoices.atlassian.net/browse/FW-5914